lib/src/config/config_file/client.example.toml

```
@@ -116,20 +112,20 @@ sync_mode = "full"
 
 ##############################################################################
 #
-# ZK Proof specific configuration
+# ZK Prover specific configuration
 #
 ##############################################################################
 #[zkp]
```

If you follow my suggestion above:
```suggestion
#[zkp_prover]
```